internal/xds/translator/utils.go

```diff
@@ -161,3 +161,17 @@ func createExtServiceXDSCluster(rd *ir.RouteDestination, tCtx *types.ResourceVer
 	}
 	return nil
 }
+
+func ParseTLSProtocol(tlsVersion ir.TLSVersion) (tlsv3.TlsParameters_TlsProtocol, error) {
```

Comment:
can we reuse the util that we have in CTP for this mapping?

Reply:
yes, i think we should be able i will check it.